src/vpp_plugins/vpp-chaperone/VppConfigManager.cpp

```diff
@@ -1644,89 +1650,96 @@ void VppConfigManager::doCpePolicerConfig (VppClient &vppClient,
   // otherwise VPP will crash if policed traffic is being passed while the
   // config is deleted.
   bool ifaceStopped = false;
-  // Delete old tables
-  std::vector<u32> tableIds;
-  vppClient.getClassifierTableIds (tableIds);
   VppClient::ClassifierTableConfig tableConfig;
-  for (const u32 &tableId : tableIds)
+  u32 tableId = ~0;
+
+  vppClient.getClassifierByIndex (interfaceIndex, tableId);
+  if (tableId != ~0)
     {
       if (!ifaceStopped)
         {
           ifaceStopped = true;
-          VLOG (1) << "Stopping interface "
-                   << interface << " before removing old policer config";
+          VLOG (1) << "Stopping interface " << interface
+                   << " before removing old policer config";
           vppClient.setInterfaceFlags (interface, false);
         }
       tableConfig.tableIndex = tableId;
       tableConfig.isAdd = 0; // Delete old table
       vppClient.addDelClassifierTable (tableConfig);
     }
-
-  // Create table
-  tableConfig.isAdd = 1; // Add
-  // Skip 14 bytes for Ethernet header, 4 bits for IPv6 version header;
-  // match on full DSCP value contained in next 6 bits
-  std::vector<u8> maskVector = {0, 0, 0, 0, 0, 0, 0,    0,
-                                0, 0, 0, 0, 0, 0, 0x0F, 0xC0};
-  tableConfig.mask = maskVector.data ();
-  vppClient.addDelClassifierTable (tableConfig);
-
+  else
+    {
+      LOG (INFO) << "Interface does not have classifier table";
```

Comment:
Use `VLOG (1)` like above, otherwise this debug log cannot be disabled

Reply:
Addressed Review Comment.